File: src/main/java/com/raoulvdberge/refinedstorage/item/StorageHousingItem.java
package com.raoulvdberge.refinedstorage.item;

import com.raoulvdberge.refinedstorage.RS;
import net.minecraft.item.Item;

public class StorageHousingItem extends Item {
    public StorageHousingItem() {
        super(new Item.Properties().group(RS.MAIN_GROUP));

        this.setRegistryName(RS.ID, "storage_housing");
    }
}

File: src/main/kotlin/com/refinedmods/refinedstorage/item/StorageHousingItem.kt
package com.refinedmods.refinedstorage.item

import com.refinedmods.refinedstorage.RS
import net.minecraft.item.Item

class StorageHousingItem : Item(Properties().group(RS.MAIN_GROUP)) {
    init {
        this.setRegistryName(RS.ID, "storage_housing")
    }
}